js/client-mainmenu.js

```diff
@@ -379,11 +379,18 @@
 			var $searchForm = $('.mainmenu button.big').closest('form');
 			var $formatButton = $searchForm.find('button[name=format]');
 			var $teamButton = $searchForm.find('button[name=team]');
-			if (this.searching) {
+			if (this.searching && (!$.isArray(this.searching) || this.searching.length)) {
```

Comment:
It seems like it would be `true` for scalars, too (e.g. `235`, `true`, or `'foobar'`).

In other words, the conditional is true unless `this.searching` is `[]`. Which... is there a clearer way to write that?